api/src/main/java/com/javadiscord/jdi/internal/api/DiscordRequestBuilder.java

```diff
@@ -130,4 +131,41 @@ protected void setSuccessResult(DiscordResponse result) {
     protected void setFailureError(Throwable ex) {
         future.setException(ex);
     }
+
+    @Override
+    public String toString() {
+        String headersString =
+            headers.entrySet().stream()
+                .map(entry -> entry.getKey() + "=" + entry.getValue())
+                .collect(Collectors.joining(", "));
+
+        String queryParamsString =
+            queryParameters.entrySet().stream()
+                .map(entry -> entry.getKey() + "=" + entry.getValue())
+                .collect(Collectors.joining(", "));
+
+        String bodyString;
+        try {
+            bodyString = OBJECT_MAPPER.writeValueAsString(body);
+        } catch (Exception e) {
```

Comment:
Why catch Exception instead of the specific one?

Reply:
Good catch! This should be a `JsonProcessingException`